compute/instances/how-to/create-an-instance.mdx

```diff
@@ -41,25 +41,25 @@ Select a tab below for instructions on how to create an Instance via either our
         - Choose an **Instance type**. Different Instance types have different prices, processing power, memory, storage options and bandwidth. See our [dedicated documentation](/compute/instances/reference-content/choosing-instance-type/) to help choose the best Instance type for your needs.
-        - Choose an **Image** to run on your Instance. This can be an operating system, an InstantApp or a custom image. A complete list of all available Linux distributions and InstantApps can be found [here](https://www.scaleway.com/en/imagehub/).
-
-        <Message type="note">
-          When choosing an image for an Enterprise Instance, be aware this Instance type is not compatible with bootscripts, except for [rescue mode](/compute/instances/how-to/use-boot-modes/#how-to-use-rescue-mode). InstantApps are also not available for this Instance type.
-        </Message>
-
+        - Choose an **Image** to run on your Instance. This can be an operating system, an InstantApp or a custom image. A complete list of all available Linux distributions and InstantApps can be found [here](https://www.scaleway.com/en/imagehub/)
         - Add **Volumes**. Volumes are storage spaces used by your Instances. 
           - For **GP1 Instances** you can leave the default settings of a maximum local storage, or choose how much [local](/compute/instances/concepts/#local-volumes) and/or [block](/compute/instances/concepts/#block-volumes) storage you want. Your **system volume** is the volume on which your Instance will boot. The system volume can be either a local or a block volume.
           - **PLAY2**, **PRO2**, and **Enterprise** Instances boot directly [on block volumes](/compute/instances/concepts/#boot-on-block). You can add several block volumes and define how much storage you want for each. 
-        
         <Message type="important">
           Note that:  
             - a volume with an OS image should be at least 10 GB (125 GB for a GPU OS).
             - if several Block Storage volumes are connected to your Instance, the first volume will contain the OS and is required to boot the Instance. You can [change your boot volume](/compute/instances/how-to/use-boot-modes/#how-to-change-the-boot-volume) after you create the Instance.
             - if you have two or more volumes with identical operating systems, or a volume with no operating system, choosing one of these as your boot volume may cause unpredictable boot behavior.
         </Message>
 
+        - Assign a public IP to the Instance.
+          - By default a public IP is assigned to your Instance. You can choose either to use a **NAT public IP** (a public IP address that uses a carrier-grate NAT to translate the Instances private IP address) or a **Routed public IP** (a deidcated public IP address routed your Instance that allows direct communication between the Instance and the Internet).
```

Comment:
```suggestion
          - By default a public IP is assigned to your Instance. You can choose either to use a **NAT public IP** (a public IP address that uses a carrier-grate NAT to translate the Instances private IP address) or a **Routed public IP** (a dedicated public IP address routed your Instance that allows direct communication between the Instance and the Internet).
```

Reply:
@bene2k1 Here, I updated it to '`routed **to** your Instance`', but as I'm not sure, I'm flagging it, so you can check, please.